python/cugraph/cugraph/tree/minimum_spanning_tree.py

```diff
@@ -87,6 +87,14 @@ def minimum_spanning_tree(
         A graph descriptor with a minimum spanning tree or forest.
         The networkx graph will not have all attributes copied over
 
+    Examples
+    --------
+    >>> M = cudf.read_csv(datasets_path / 'netscience.csv', delimiter='\t',
+    ...                   dtype=['int32', 'int32', 'float32'], header=None)
+    >>> G = cugraph.Graph()
+    >>> G.from_cudf_edgelist(M, source='0', destination='1')
+    >>> # cugraph.minimum_spanning_tree(G)
```

Comment:
Is this and the algo call in the example below broken, or just an oversight?  If this isn't working in doctest, can you add a FIXME? If the algos themselves are broken, can you file an issue?